docs/site/Model.md

```diff
@@ -116,8 +116,9 @@ export class Customer {
 
 Models can be created at runtime using the `defineModelClass()` helper function
 from the `@loopback/repository` class. It expects a base model to extend
-(typically `Model` or `Entity`), folowed by the model definition object as shown
-in the example below.
+(typically `Model` or `Entity`), followed by a `ModelDefinition` object as shown
+in the example below. The `ModelDefinition` object is an abstraction for
+specifying the various attributes of a LoopBack model.
```

Comment:
Sorry for not being clear enough.

> Can you please explain what is the second type parameter for?

In my comment ☝️, I was referring to the second item in the list of generic parameters for `defineModelClass`, i.e. the inline anonymous interface `{id: number; title?: string}`.

Now that I am looking at the example below again, I think may be good to show how `bookDef` is created too.

```ts
const bookDef = new ModelDefinition('Book')
  .addProperty('id', {type: 'number', id: true})
  .addProperty('title', {type: 'string'});
const BookModel = // ...
```

Reply:
I am afraid this comment is still not addressed.

As a user not familiar with defineModelClass, I don't understand what properties it should describe and why. Where is id and title coming from, why they have the particular type (number, string) and why title is optional while id is required?

I am looking for an explanation that
- the interface is providing TypeScript description for the properties of the model we are defining
- when omitted, the class will have only members inherited from the base model class, which typically means no properties